src/main/java/StringCalculator.java

```diff
@@ -0,0 +1,51 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+    private static List<String> delimeter = new ArrayList<>(List.of(":", ","));
+
+    public static int calculate(String input) {
+        if (input == null || input.isEmpty()) {
```

Comment:
```suggestion
        if (isBlank()) {
```
if body 영역은 메서드로 분리하는 것은 어떨까?